teps/OWNERS

```diff
@@ -0,0 +1,32 @@
+approvers:
+- ImJasonH
+- PuneetPunamiya
+- a-roberts
+- afrittoli
+- alangreene
+- bobcatfish
+- chetan-rns
+- chmouel
+- danielhelfand
+- dibbles
+- dibyom
+- dlorenc
+- houshengbo
+- hrishin
+- iancoffey
+- kimsterv
+- mnuttall
+- ncskier
+- nikhil-thomas
+- piyush-garg
+- pradeepitm12
+- pratap0007
+- sbwsg
+- skaegi
+- sm43
+- steveodonovan
+- sthaha
+- vdemeester
+- vincent-pli
+- vtereso
+- wlynch
```

Comment:
i often wish we could `@` tag project owners in PRs 🤔 i wonder if we could eventually have some automation that creates github groups from OWNERS files

maybe related: can we refer to aliases here instead of individual names?

Reply:
> i often wish we could `@` tag project owners in PRs thinking i wonder if we could eventually have some automation that creates github groups from OWNERS files

That should be possible.

> maybe related: can we refer to aliases here instead of individual names?

That should be possible too, but we would need to define the aliases there too, so it doesn't change *too much* :stuck_out_tongue:

Reply:
perhaps has a follow-up PR? I think it'd be more readable / maintainable